label_studio/projects/serializers.py

```diff
@@ -94,6 +94,7 @@ def to_internal_value(self, data):
         # FIXME: remake this logic with start_training_on_annotation_update
         initial_data = data
         data = super().to_internal_value(data)
+
```

Comment:
super nit: as long as we're making small changes, let's remove the blank line

Reply:
we have such separation of vars and code in frontend maybe it's the same? I'm going to resolve this anyway to merge PR